app/src/main/java/org/ieeeguc/ieeeguc/fragments/user/UserShow.java

```diff
@@ -6,19 +6,61 @@
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.TextView;
 
+import com.google.gson.Gson;
+
+import org.ieeeguc.ieeeguc.HTTPResponse;
 import org.ieeeguc.ieeeguc.R;
+import org.ieeeguc.ieeeguc.models.User;
+import org.json.JSONObject;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
 
 /**
  * A simple {@link Fragment} subclass.
  */
 public class UserShow extends Fragment {
 
+    private User user;
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_user_show, container, false);
+
+
+        View view = inflater.inflate(R.layout.fragment_user_show, container, false);
+        int id = new Gson().fromJson(getArguments().getString("user_id"),Integer.class);
+        String token  = new Gson().fromJson(getArguments().getString("token"),String.class);
+        User.getUser(token, id, new HTTPResponse() {
+            @Override
+            public void onSuccess(int statusCode, JSONObject body) {
+
+            }
+
+            @Override
+            public void onFailure(int statusCode, JSONObject body) {
+
+            }
+        });
+        ((TextView) view.findViewById(R.id.name)).setText(user.getFirstName()+" "+user.getLastName());

```

Comment:
The above code `User.getUser()` is Async!! so this statement will run in parallel with it!!!